openquake/hazardlib/gsim/lanzano_2019.py

```diff
@@ -67,9 +67,7 @@ def _site_amplification(ctx, C):
     it is given by FS = klog10(V0/800) , where V0 = Vs30 when Vs30 <= 1500
     and V0=1500 otherwise
     """
-    v0 = np.ones_like(ctx.vs30) * 1500.
-    v0[ctx.vs30 < 1500] = ctx.vs30
-    return C['k'] * np.log10(v0/800)
+    return C['k'] * np.log10(np.clip(ctx.vs30, -np.inf, 1500.0) / 800.0)
```

Comment:
Thanks. Indeed we also noticed this problem. If possible, it would be worth having a test for this (even just one line in a verification table).